dallinger/frontend/templates/login.html

```diff
@@ -0,0 +1,26 @@
+{% extends "base/dashboard.html" %}
+
+{% block body %}
+    <h1>Sign In</h1>
+    <form action="" method="post" novalidate>
+        {{ form.hidden_tag() }}
+        <p>
+            {{ form.username.label }}<br>
+            {{ form.username(size=32) }}<br>
+            {% for error in form.username.errors %}
+            <span style="color: red;">[{{ error }}]</span>
+            {% endfor %}
+        </p>
+        <p>
+            {{ form.password.label }}<br>
+            {{ form.password(size=32) }}<br>
+            {% for error in form.password.errors %}
+            <span style="color: red;">[{{ error }}]</span>
+            {% endfor %}
+        </p>
+        <p>{{ form.remember_me() }} {{ form.remember_me.label }}</p>
```

Comment:
Yeah, that was just me following the tutorial. We can probably remove it, but some experiments run over the course of multiple days, so it may be helpful to be able to ask for a persistent login session.

Reply:
Is the cookie specific to an experiment, or will I be able to access _any_ dallinger dashboard once I have the cookie?

Reply:
It’s specific to an experiment run. The Flask `SECRET_KEY` changes with every run, as does the password. Now that I think about it, the way it’s set now might be a problem because the key will vary from web dyno to web dyno. If Heroku has sticky session bars routing that’s no issue, but otherwise it may be an issue for experiments with multiple web dynos.